.vscode/tasks.json

```diff
@@ -11,7 +11,7 @@
         "/property:GenerateFullPaths=true",
         "/consoleloggerparameters:NoSummary"
       ],
-      "problemMatcher": "$msCompile"
+      "problemMatcher": []
```

Comment:
do you know what this change is about?

Reply:
I should have talk about this one. So OmniSharp will report diagnostics in the Problems tab. Running a build using a problems matcher will report build warnings and errors in the Problemss tab. This will duplicate warning and errors in the Problems tab amd the build reported problems won't go away until a new build is run. So, removing the problemMatcher will keep duplicate, zombie issues from entering the Problems tab.